From patches Mon Sep 17 00:00:00 2001
From: "" <mrhines@linux.vnet.ibm.com>
To: quintela@redhat.com
Cc: Anthony Liguori <aliguori@us.ibm.com>, qemu-devel@nongnu.org,
 Michael R Hines <mrhines@us.ibm.com>, Bulent Abali <abali@us.ibm.com>,
 owasserm@redhat.com, pbonzini@redhat.com, Gokul B Kandiraju <gokul@us.ibm.com>,
 eblake@redhat.com
Date: Wed, 24 Apr 2013 15:00:50 -0400
Message-id: 1366830057-16964-6-git-send-email-mrhines@linux.vnet.ibm.com
Subject: [PATCH 05/11] rdma: export qemu_fflush()

From: "Michael R. Hines" <mrhines@us.ibm.com>

RDMA uses this to flush the control channel before sending its
own message to handle page registrations.

Signed-off-by: Michael R. Hines <mrhines@us.ibm.com>
---
 include/migration/qemu-file.h |    1 +
 savevm.c                      |    2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/migration/qemu-file.h b/include/migration/qemu-file.h
index 7e097c1..97d4277 100644
--- a/include/migration/qemu-file.h
+++ b/include/migration/qemu-file.h
@@ -111,6 +111,7 @@ void qemu_file_reset_rate_limit(QEMUFile *f);
 void qemu_file_set_rate_limit(QEMUFile *f, int64_t new_rate);
 int64_t qemu_file_get_rate_limit(QEMUFile *f);
 int qemu_file_get_error(QEMUFile *f);
+void qemu_fflush(QEMUFile *f);
 
 static inline void qemu_put_be64s(QEMUFile *f, const uint64_t *pv)
 {
diff --git a/savevm.c b/savevm.c
index 28334e7..7e6e741 100644
--- a/savevm.c
+++ b/savevm.c
@@ -586,7 +586,7 @@ static inline bool qemu_file_is_writable(QEMUFile *f)
  * If there is writev_buffer QEMUFileOps it uses it otherwise uses
  * put_buffer ops.
  */
-static void qemu_fflush(QEMUFile *f)
+void qemu_fflush(QEMUFile *f)
 {
     ssize_t ret = 0;
 
-- 
1.7.10.4

